packages/adblocker/src/engine/engine.ts

```diff
@@ -283,7 +409,7 @@ export default class FilterEngine extends EventEmitter<EngineEventHandlers> {
       buffer.pos = currentPos;
     }
 
-    const engine = new this({ config });
+    const engine = new this({ config, version: serializedEngineVersion });
```

Comment:
Should this be reverted?

Reply:
Thanks for pointing, fixed in [`52a07ac` (#4138)](https://github.com/ghostery/adblocker/pull/4138/commits/52a07ac5670f50d05dd602dff7e56ced9f772170)